DOC_CHANGES.md

```diff
@@ -0,0 +1,10 @@
+<!---
+This file is reset everytime when a new release is done. Contents of this file is for the currently unreleased version.
+-->
+
+# win32-certstore doc changes
+
+Documentation changes are given below for **win32-certstore version 0.1.0**.
+
+## `win32-certstore` subcommand changes
```

Comment:
you mentioned that doc changes are given below but it does not reflecting anything.